tests/models/test_serialized_dag.py

```diff
@@ -117,7 +116,6 @@ def test_remove_stale_dags(self):
         self.assertFalse(SDM.has_dag(stale_dag.dag_id))
         self.assertTrue(SDM.has_dag(fresh_dag.dag_id))
 
-    @mock.patch('airflow.models.serialized_dag.STORE_SERIALIZED_DAGS', True)
     def test_bulk_sync_to_db(self):
```

Comment:
How does this test work now, this doesn't store serialized DAG anymore, does it ?

Reply:
I mean with the patch, it would only sync and write DAGs to DAGModel table and assert that queries count is 7, is that the intention

Reply:
`SDM.bulk_sync_to_db` will now _always_ write to the DB when called.

Before it would return if the config was false. Now  the check is moved in to DagBag.bulk_sync:

```python
        if self.store_serialized_dags:
            SerializedDagModel.bulk_sync_to_db(self.dags.values())
```

Reply:
(This test isn't great, but that wasn't changed by this)